base/dotenv.mustache

```diff
@@ -2,4 +2,4 @@ JWT_SECRET=foo
 RETHINK_HOST=localhost
 RETHINK_PORT=28015
 RETHINK_NAME={{name}}
-PORT=3001
+PORT=4678
```

Comment:
@s-taylor was this changed for a reason? The commit message said "to prevent clashing"